cloud/gcp/udmi-sites.tf.template

```diff
@@ -1,8 +1,8 @@
 module "ZZ-TRI-FECTA" {
   source      = "./modules/terraform-google-udmi-site"
   gcp_project = var.gcp_project_name
-  site_name   = "ZZ-TRI-FECTA"
-  site_region = "europe-west1"
-  site_group  = "group:group@example.com"
+  site_name   = "@UDMI_SITE_NAME@"
+  site_region = "@UDMI_SITE_REGION@"
+  site_group  = "group:@UDMI_SITE_GROUP@"
```

Comment:
This group account is to give only viewer/provisioning access to IoT Core registries and subscriber access to the pub/sub topic resources, so it should have a different variable than the one in the variables template file

Reply:
done